assets/config/log4j2.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration packages="org.apache.logging.log4j.core,io.sentry.log4j2">
+    <Appenders>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout pattern="[REPORT] %d{ddMMyyyy HH:mm:ss,SSS} %-5p [%t] [%c{1}] %m%n"/>
+        </Console>
+        <File name="ReportingRollingLog" fileName="opensrpReporting.log">
+            <PatternLayout pattern="%d{ddMMyyyy HH:mm:ss,SSS} %-5p [%t] [%c{1}] %m%n"/>
+        </File>
+        <File name="ReportingMonitorLog" fileName="opensrpReportingMonitoring.log">
+            <PatternLayout pattern="%d{ddMMyyyy HH:mm:ss,SSS} %m%n"/>
+        </File>
+        <File name="WebRollingLog" fileName="opensrpWeb.log">
+            <PatternLayout pattern="%d{ddMMyyyy HH:mm:ss,SSS} %-5p [%t] [%c{1}] %m%n"/>
+        </File>
+        <File name="WebMonitorLog" fileName="opensrpWebMonitoring.log">
+            <PatternLayout pattern="%d{ddMMyyyy HH:mm:ss,SSS} %m%n"/>
+        </File>
+        <Sentry name="Sentry"/>
```

Comment:
are we including all lo levels?

Reply:
also not sure how is the connection to sentry defined, what happens to dev environments where we may not want to send logs to sentry.

Reply:
Nope by default Sentry sends `Error` and above. If the `dsn` is an empty string then the Sentry SDK is closed

Reply:
https://docs.sentry.io/platforms/java/guides/log4j2/#configure